File: 2185-CountingWordsWithAGivenPrefix/2185-CountingWordsWithAGivenPrefix.py
# Last updated: 07/09/2026, 22:48:59
class Solution:
    def prefixCount(self, words: List[str], pref: str) -> int:
        result = 0
        for word in words:
            if (word.startswith(pref)):
                result += 1

        return result

File: 2185-CountingWordsWithAGivenPrefix/2185-CountingWordsWithAGivenPrefix.cpp
// Last updated: 07/09/2026, 22:48:51
class Solution {
public:
    int prefixCount(vector<string>& words, string pref) {
        const int prefix_size = pref.size();
        int result = 0;
        for (const string& word : words){
            result += (word.substr(0, prefix_size) == pref);
        }
        return result;
    }
};